core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -5841,6 +5841,53 @@ private HepProgram getTransitiveProgram() {
         .check();
   }
 
+  /** Tests use of a measure in a non-aggregate query. Calls
+   * {@link RelOptFixture#checkUnchanged()} because Sql-to-rel has done the
+   * necessary work already. */
```

Comment:
Would it be better to add jira link?

Reply:
No, because this query has no `ORDER BY` such queries were working before this change.